### PR TITLE
DOCSP-39854: Remove async-std runtime support

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-rust/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-39854
Staging - https://preview-mongodbnorareidy.gatsbyjs.io/rust/DOCSP-39854-tokio-runtime/fundamentals/runtimes/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/fundamentals/runtimes.txt [hunk 1]
```diff
@@ -14,47 +14,37 @@ Overview
...
-      .. code-block:: toml
-
-         [dependencies.mongodb]
-         version = "{+version+}"
-         features = ["tokio-runtime"]
-
-   .. tab:: async-std
-      :tabid: asynchronous-api-async-std
+The driver uses the ``tokio`` runtime by default, so you can use this runtime without specifying any
+feature flags in your project's ``Cargo.toml`` file. You can also explicitly choose the ``tokio`` runtime
```
Comment (jordan-smith721) [inline source/fundamentals/runtimes.txt:39]: Question: Is there a reason they might want to do this still if tokio is the default? If not, I wonder if we can just remove the code example to simplify things.

File: source/fundamentals/runtimes.txt [hunk 2]
```diff
@@ -14,47 +14,37 @@ Overview
...
-You can select the synchronous API by adding feature flags to the ``mongodb`` dependency in your ``Cargo.toml`` file.
+.. important::
+
+   Beginning in {+driver-short+} v3.0, the ``tokio`` runtime is the only asynchronous runtime crate that
+   the driver supports. Previous versions of the driver also support the ``async-std`` runtime crate. To
+   enable the ``async-std`` runtime, you must use a version of the {+driver-short+} that precedes v3.0.
+
+The driver also includes a synchronous API for use cases that require blocking, or when parallelism
+is not necessary. You can select the synchronous API by adding feature flags to the ``mongodb`` dependency
+in your ``Cargo.toml`` file.
```
Comment (jordan-smith721) [inline source/fundamentals/runtimes.txt:31]: Is it shown how to do this somewhere, or is this standard practice for Rust users so they should already know how to do it? Just curious if an example would be needed here.

**Review by jordan-smith721 — APPROVED** [review]
LGTM w/ two questions

File: source/fundamentals/runtimes.txt (hunk 2 above)
Comment (norareidy) [reply source/fundamentals/runtimes.txt:31]: There's an example of this in the "Configure the Synchronous API" section of the page! I added a list of all the page sections just to clarify what's included in the guide (which we usually do for rust guides anyway)

File: source/fundamentals/runtimes.txt (hunk 1 above)
Comment (norareidy) [reply source/fundamentals/runtimes.txt:53]: The main reason would be to use both the synchronous and asynchronous runtimes in the same project. We do have a section on this page that shows how to enable both, so I removed the code example and just linked to that section instead

File: source/fundamentals/runtimes.txt [hunk 3]
```diff
@@ -14,50 +14,51 @@ Overview
...
+  ``tokio`` asynchronous runtime
 
-.. tabs::
+- :ref:`rust-runtimes-configure-sync` shows how to configure your application to use the
+  ``sync`` and ``tokio-sync`` synchronous runtimes
 
-   .. tab:: tokio
-      :tabid: asynchronous-api-tokio
+- :ref:`rust-runtimes-configure-both` shows how to enable both the asynchronous and synchronous
+   runtimes APIs in your application
```
Comment (jordan-smith721) [inline source/fundamentals/runtimes.txt:42]: I think there's an extra space on the second line here that's causing the bullet to render weird

```suggestion
- :ref:`rust-runtimes-configure-both` shows how to enable both the asynchronous and synchronous
  runtimes APIs in your application
```

File: source/fundamentals/runtimes.txt [hunk 4]
```diff
@@ -14,50 +14,48 @@ Overview
...
+runtime.
 
-The driver also includes a synchronous API for use cases that require blocking, or when parallelism is not necessary.
-You can select the synchronous API by adding feature flags to the ``mongodb`` dependency in your ``Cargo.toml`` file.
+.. important::
 
-.. _rust-runtimes-configure-async:
+   Beginning in {+driver-short+} v3.0, the ``tokio`` runtime is the only asynchronous runtime crate that
+   the driver supports. Previous versions of the driver also support the ``async-std`` runtime crate. To
+   enable the ``async-std`` runtime, you must use a version of the {+driver-short+} that precedes v3.0.
```
[the remark this answers is not in the document]
Comment (norareidy) [reply source/fundamentals/runtimes.txt]: Good to know - I added this info to the admonition, let me know what you think

File: source/fundamentals/runtimes.txt [hunk 5]
```diff
@@ -23,8 +23,11 @@ runtime.
 .. important::
 
    Beginning in {+driver-short+} v3.0, the ``tokio`` runtime is the only asynchronous runtime crate that
-   the driver supports. Previous versions of the driver also support the ``async-std`` runtime crate. To
-   enable the ``async-std`` runtime, you must use a version of the {+driver-short+} that precedes v3.0.
+   the driver supports. Previous versions of the driver also support the ``async-std`` runtime crate.
+   
+   If your application uses the ``async-std`` runtime, you can start a ``tokio`` runtime in the same application
+   for {+driver-short+} operations. Specify the ``#[tokio::main]`` attribute above functions that run
```
[the remark this answers is not in the document]
Comment (norareidy) [reply source/fundamentals/runtimes.txt]: Ah okay, I reworded and linked to that API docs example

**Review by abr-egn — APPROVED** [review]
LGTM!

**Pull request merged**